tools/webpack

```diff
@@ -5,82 +5,95 @@ import os
 import subprocess
 from typing import NoReturn
 
-os.chdir(os.path.join(os.path.dirname(__file__), '..'))
-STATIC_PATH = 'static/'
+os.chdir(os.path.join(os.path.dirname(__file__), ".."))
+STATIC_PATH = "static/"
 
 
 def build_for_prod_or_casper(quiet: bool) -> NoReturn:
     """Builds for production, writing the output to disk"""
 
-    webpack_args = ['node', 'node_modules/.bin/webpack-cli',
-                    '--config', 'tools/webpack.config.ts', '-p',
-                    '--env', 'production']
+    webpack_args = [
+        "node",
+        "node_modules/.bin/webpack-cli",
+        "--config",
+        "tools/webpack.config.ts",
+        "-p",
+        "--env",
+        "production",
+    ]
     if quiet:
-        webpack_args += ['--display', 'errors-only']
+        webpack_args += ["--display", "errors-only"]
     os.execvp(webpack_args[0], webpack_args)
 
+
 def build_for_dev_server(host: str, port: str, minify: bool, disable_host_check: bool) -> None:
     """watches and rebuilds on changes, serving files from memory via webpack-dev-server"""
 
     # This is our most dynamic configuration, which we use for our
     # dev server.  The key piece here is that we identify changes to
     # files as devs make edits and serve new assets on the fly.
-    webpack_args = ['node', 'node_modules/.bin/webpack-dev-server']
+    webpack_args = ["node", "node_modules/.bin/webpack-dev-server"]
     webpack_args += [
-        '--config',
-        'tools/webpack.config.ts',
+        "--config",
+        "tools/webpack.config.ts",
         # webpack-cli has a bug where it ignores --watch-poll with
         # multi-config, and we don't need the katex-cli part anyway.
-        '--config-name', 'frontend',
-        '--allowed-hosts', ','.join([host, '.zulipdev.com', '.zulipdev.org']),
-        '--host', host,
-        '--port', port,
+        "--config-name",
+        "frontend",
+        "--allowed-hosts",
+        ",".join([host, ".zulipdev.com", ".zulipdev.org"]),
+        "--host",
+        host,
+        "--port",
+        port,
```

Comment:
I regret that the formatter splits the argument name/value pairs here; is there a good workaround?

Reply:
`f"--host={host}"`